tests/test_project.py

```diff
@@ -410,8 +416,8 @@ def test_context(job):
         project = A(self.mock_project().config)
         for job in project:
             job.doc.np = 3
-            next_op = project.next_operation(job)
-            assert 'mpirun -np 3 python' in next_op.cmd
+            next_op = project.next_operations(job)
```

Comment:
We haven't removed support for `next_operation` yet. We need to continue testing the deprecated feature until it is removed. You should add a warning filter and potentially add a second test that uses `next_operations`.